erpnext/patches.txt

```diff
@@ -730,4 +730,5 @@ execute:frappe.delete_doc_if_exists("Report", "Item-Wise Sales Register")
 execute:frappe.delete_doc_if_exists("Report", "Sales Analytics")
 erpnext.patches.v13_0.update_accounts_taxjar_setting #
 erpnext.patches.v13_0.rename_task_cancelled_status_to_closed
-erpnext.patches.v13_0.add_default_project_into_project_details_child_table
\ No newline at end of file
+erpnext.patches.v13_0.add_default_project_into_project_details_child_table
+erpnext.patches.v13_0.rename_project_field_to_default_project
```

Comment:
rename_project_field_to_default_project needs to be executed before add_default_project_into_project_details_child_table